config.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget android-versionCode="535" id="uk.ac.ucl.ihi.onthemove" ios-CFBundleIdentifier="uk.ac.ucl.ihi.onthemove" ios-CFBundleVersion="1" version="0.7.3" xmlns:android="http://schemas.android.com/apk/res/android">
+<widget android-versionCode="536" id="uk.ac.ucl.ihi.onthemove" ios-CFBundleIdentifier="uk.ac.ucl.ihi.onthemove" ios-CFBundleVersion="1" version="0.7.3" xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
please merge with the correct bundle identifier and version for iOS.

Reply:
Ok.